NCS.DSS.EmploymentProgression/PostEmploymentProgression/Service/EmploymentProgressionPostTriggerService.cs

```diff
@@ -9,10 +9,10 @@ namespace NCS.DSS.EmploymentProgression.PostEmploymentProgression.Service
 {
     public class EmploymentProgressionPostTriggerService : IEmploymentProgressionPostTriggerService
     {
-        private readonly IDocumentDBProvider _documentDbProvider;
-        private readonly IServiceBusClient _serviceBusClient;
+        private readonly ICosmosDBProvider _documentDbProvider;
```

Comment:
Can we rename this to cosmosDBProvider please as we no longer use documentDBProvider